test/unit/init/init_test.js

```diff
@@ -87,6 +84,14 @@ suite('init', () => {
       });
     });
 
+    test('works with default env', () => {
```

Comment:
The not found case is covered in another test, checking not found is an implementation detail of this test.